.werft/platform-trigger-artificial-job.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+#
+# The easiest way to run this script is through Werft so you don't have to worry
+# about installing the appropraite service account etc. locally.
+#
+#   werft job run github -j .werft/platform-trigger-artificial-job.yaml -s .werft/platform-trigger-artificial-job.sh
+#
+
+sleep 1
+
+set -Eeuo pipefail
+
+# Activate service account and install core-dev context
+gcloud auth activate-service-account --key-file "/mnt/secrets/gcp-sa/service-account.json"
+gcloud container clusters get-credentials core-dev --zone europe-west1-b --project gitpod-core-dev
+
+echo "[Creating new branch|PHASE] Trigger new job with preview environment"
+echo "[Creating new branch] Setting up user config"
+git config --global user.name roboquat
+git config --global user.email roboquat@gitpod.io
+git remote set-url origin https://oauth2:"${ROBOQUAT_TOKEN}"@github.com/gitpod-io/gitpod.git
```

Comment:
I think this fails because you're missing a `sudo chown -R gitpod:gitpod /workspace` - at least all other jobs seems to have this ☺️